### ISSUE TITLE
Not the same old map tools...& live link is dead

### ISSUE THREAD

**Issue by upcamogie** [Esri/map-tools-template#16]

**Comment by kellyhutchins** [comment]
This is the correct app. I've updated the 'View it live' link.

**Comment by upcamogie** [comment]
thanks, i am seeing that now. is there a way to config a proxy like other apps? i haven't looked too closely yet,

**Comment by kellyhutchins** [comment]
Yep you can setup the proxy then specify the proxy url in the config/defaults file.
